src/test/java/io/zipcoder/casino/CardTest.java

```diff
@@ -118,7 +118,11 @@ public void comparingCards()
         Integer expectedFirstCompare = 1;
         Integer expectedSecondCompare = -1;
         Integer expectedThirdCompare = 1;
+<<<<<<< HEAD
+
```

Comment:
Something funky here in the test area, probably a git issue. Take these out first, then push again and do another pull request.